doc/api/http2.md

```diff
@@ -2673,6 +2673,16 @@ added: v10.1.0
 The `request.aborted` property will be `true` if the request has
 been aborted.
 
+#### request.complete
```

Comment:
nit: have the feeling it s in alphabetical order isn't it ?

Reply:
It wasn't it, but now is fixed.